guaranteed-message-delivery/README.md

```diff
@@ -0,0 +1,390 @@
+# Guaranteed Message Delivery
+
+## Overview
+
+In this tutorial, you will create a service that let users reserve appointments at a hospital. To manage the flow of requests, you will employ a message broker. The publisher service will retrieve the appointment requests and forward them to the message broker. The consumer service will listen to the broker's message queue and interact with the hospital's backend and SMS service to finalize the reservation.
+
+To implement this use case, you will develop a REST service with a single resource using Visual Studio Code with the Ballerina Swan Lake extension. This resource will handle incoming user requests and forward them to the message broker. The consumer service that listens to the message broker's queue for new appointment requests will trigger a backend call to the hospital to make the reservation and send an SMS to the patient's phone number.
```

Comment:
A broker obviously can have many queues. I would rephrase similar to the previous para.

Reply:
The second sentence is kinda repetitive anyway. This para is focused more on how exactly the development will happen.